tests/protractor/e2e/send-message.js

```diff
@@ -0,0 +1,223 @@
+var utils = require('../utils'),
+    _ = require('underscore');
+
+describe('Send message', function() {
+  'use strict';
+
+  var RAW_PH = '+447765902000';
+
+  var ALICE = {
+    _id: 'alice-contact',
+    type: 'person',
+    name: 'Alice Alison',
+    phone: '+447765902001'
+  };
+  var BOB_PLACE = {
+    _id: 'bob-contact',
+    type: 'clinic',
+    name: 'Bob Place'
+  };
+  var CAROL = {
+    _id: 'carol-contact',
+    type: 'person',
+    name: 'Carol Carolina',
+    parent: BOB_PLACE
+  };
+  var DAVID = {
+    _id: 'david-contact',
+    type: 'person',
+    name: 'David Davidson',
+    phone: '+447765902002',
+    parent: BOB_PLACE
+  };
+
+  var CONTACTS = [ALICE, BOB_PLACE, CAROL, DAVID];
+
+  var savedUuids = [];
+  beforeAll(function(done) {
+    browser.ignoreSynchronization = true;
+    protractor.promise
+      .all(CONTACTS.map(utils.saveDoc))
+      .then(function(results) {
+        results.forEach(function(result) {
+          savedUuids.push(result.id);
+        });
+        done();
+      })
+      .catch(function(err) {
+        console.error('Error saving docs', err);
+        done();
+      });
+  });
+
+  afterAll(function(done) {
+    protractor.promise
+      .all(savedUuids.map(utils.deleteDoc))
+      .then(function() {
+        return utils.requestOnTestDb({
+          path: '/_design/medic/_view/tasks_messages',
+          method: 'GET'
+        });
+      })
+      .then(function(results) {
+        var ids = _.uniq(_.pluck(results.rows, 'id'));
+        return protractor.promise.all(ids.map(utils.deleteDoc));
+      })
+      .then(done);
+  });
+
+
+  var messageInList = function(identifier) {
+    return '#message-list li[data-record-id="'+identifier+'"]';
+  };
+
+  var smsMsg = function(key) {
+    return 'Hello ' + key + ' this is a test SMS';
+  };
+
+  it('can send messages to raw phone numbers', function() {
+
+    element(by.id('messages-tab')).click();
+
+    expect(element.all(by.css(messageInList(RAW_PH))).count()).toBe(0);
+
+    var sendMessageModal = element(by.id('send-message'));
+
+    expect(element(by.id('messages-send-message')).isDisplayed()).toBeTruthy();
+    expect(sendMessageModal.isDisplayed()).toBeFalsy();
+
+    element(by.id('messages-send-message')).click();
+    browser.sleep(1000);
```

Comment:
I've created https://github.com/medic/medic-webapp/issues/2660 to work on this. If you're OK with it I'd rather this stuff got merged without that, because it does work, it's just not ideal. I plan to work on #2600 straight away, I'd just rather not keep this branch a branch.